### PR TITLE
EZP-29827: Harmonized Preview Switcher in Content Section Editing view

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-29827](https://jira.ez.no/browse/EZP-29827)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [X] Ready for Code Review

ez platform 70
ez platform 71
ez platform 72

### CODE REVIEW

File: src/bundle/Resources/public/scss/_preview.scss [hunk 1]
```diff
@@ -54,14 +56,25 @@
...
             .ez-preview__action {
-                line-height: 0;
-                display: block;
-                padding: .5rem;
-                border: none;
-                border-radius: initial;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                flex: 0 0 34px;
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_preview.scss]: Maybe `flex: 0 0 calculateRem(34px)`?

File: src/bundle/Resources/public/scss/_preview.scss [hunk 2]
```diff
@@ -76,32 +89,31 @@
...
             margin: 0 auto;
-            transition: all .2s $ez-admin-transition;
+            transition: all 0.2s $ez-admin-transition;
         }
 
         &--tablet {
             background: url('/bundles/ezplatformadminui/img/preview-tablet.png') top center no-repeat;
-            height: 1250px;
-            margin-top: 1rem;
+            height: calculateRem(1250px);
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_preview.scss]: I'm wondering, do we need such big value for `height`? I mean, it will cause the scroll.

Comment (inakijv) [reply src/bundle/Resources/public/scss/_preview.scss]: Hi @sunpietro, I'd say yes if we want to show the full `preview-tablet.png` img which is preview's objective. In any case given that you are pointing this out `preview-tablet.png` height is 1287px, hence maybe the correct height would be 1290px.

**Comment by inakijv** [comment]
Updated based on review, and added fixup for responsiveness in order to avoid behavior of an enlarged switcher when resizing browser window (see screenshot of behavior detected and corrected)
screen shot 2018-11-26 at 11 07 56 am

**Comment by inakijv** [comment]
Ping @sunpietro

**Pull request merged**